Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -823,7 +827,8 @@ def condition(self, fragment, stepList, key, hasHarvest):
             ('2018' in key and fragment == "ZMM_13"),
             ('2021' in key and fragment == "ZMM_14" and 'FS' not in key),
             ('2023' in key and fragment == "ZMM_14" and 'FS' not in key),
-            ('2026D88' in key and fragment == "TTbar_14TeV" and "PixelOnly" in self.suffix)
+            ('2026D88' in key and fragment == "TTbar_14TeV" and "PixelOnly" in self.suffix),
```

Comment:
Do we need to specific on D88 only? Currently, the baseline is D95 and will move to D98 in the next pre-release.

We can keep it with this PR, and change at the next small PR also.